reana/reana_dev/git.py

```diff
@@ -329,6 +329,56 @@ def git_fork(component, exclude_components, browser):  # noqa: D301
     )
 
 
+@click.option(
+    "--component",
+    "-c",
+    multiple=True,
+    default=["CLUSTER"],
+    help="Which components? [shortname|name|.|CLUSTER|ALL]",
+)
+@click.option(
+    "--exclude-components",
+    default="",
+    help="Which components to exclude? [c1,c2,c3]",
+)
+@git_commands.command(name="git-fork-automatic")
+def git_fork_automatic(component, exclude_components):  # noqa: D301
```

Comment:
Maybe we can merge this command in `git-fork` by creating a new flag? There is already `-b`/`--browser`, so we could add `--gh`? What do you think?

Reply:
Nice idea! I added `--automatic`, that I think hints better at the purpose of the flag.